doc/manuals/devel/cookbook.md

```diff
@@ -106,16 +106,15 @@ Note the right-most column saying `(environment variable)` for the `-fg` option.
 [Top](#top)
```

Comment:
The new module should be added to the list at https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/sourceCode.md#source-code-directory-listing, with its corresponding brief description in that file, isn't it?

Reply:
Fixed in d65b3f927